packages/plugins/Pins/README.md

```diff
@@ -23,13 +23,39 @@ The usage of `displayComponent` has no influence on the creation of Pins on the
 | toZoomLevel | number? | Zoom level to use on outside input by e.g. address search. Defaults to `0`. |
 
 
+Example configuration:
+```js
+pins: {
```

Comment:
Please add an example for `coordinateSource`

Reply:
https://github.com/Dataport/polar/pull/158/commits/715eb391142416d0734db1756355c58cbfec61f5